testing/libfixposix/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+pkgname=libfixposix
+pkgver=0.4.3
+pkgrel=1.0
+pkgdesc="Thin wrapper over POSIX syscalls"
+url="https://github.com/sionescu/libfixposix"
+arch="all"
+license="BSL-1.0"
+makedepends="autoconf automake libtool"
+source="$pkgname-$pkgver.tar.gz::https://github.com/sionescu/libfixposix/archive/v$pkgver.tar.gz"
+
+prepare () {
+	   cd "$builddir"
+	   autoreconf -f -i
+}
+
+check () {
+      	 cd "$builddir"
```

Comment:
indentation is wrong, it should be 1 tab

Reply:
I'm really confused why it's showing that way. In your review comment it looks like two tabs, but when I go to look at the source it only shows as a single tab.

Reply:
Your text editor is probably misconfigured 
![image](https://user-images.githubusercontent.com/30738253/63469951-c9426200-c441-11e9-9e34-16abd26b84c1.png)

Reply:
Yeah that's definitely it, it's creating tab with 8 characters rather than 4. Which I believe was my confusion. Fix incoming, then I'll figure out what I broke with emacs..